README.md

```diff
@@ -11,6 +11,10 @@ This is the backend client of Bridge-In-Tech.
 
 Please read our [Contributing Guidelines](.github/CONTRIBUTING.md), [Code of Conduct](.github/CODE_OF_CONDUCT.md) and [Reporting Guidelines](.github/REPORTING_GUIDELINES.md) thoroughly.
 
+**BridgeInTech Environment Setup Instructions**
+
+https://github.com/anitab-org/bridge-in-tech-backend/wiki/BIT-development-environment-setup
```

Comment:
It's good that you have now placed it in a separate section @mahakbansal2019. However, instead of showing it as http url, can you mask it with a descriptive link (like the one in `Contributing` section) perhaps say something like...
"To start contributing to the project, setup the backend environment on your local machine by following the instructions on the **BIT Development Environment Setup Instruction** wiki page" 
(on the bold words you can place that url link to wiki). Does this make sense?

Reply:
Okay!
Will make the desired changes :)